content/en/docs/ops/diagnostic-tools/istioctl/index.md

```diff
@@ -101,23 +101,40 @@ See [Debugging Envoy and Istiod](/docs/ops/diagnostic-tools/proxy-cmd/) for more
 
 {{< tab name="macOS" category-value="macos" >}}
 
-If you are using the macOS operating system with the Bash terminal shell, make sure that the `bash-completion` package is installed. With the [brew](https://brew.sh) package manager for macOS, you can check to see if the `bash-completion` package is installed with the following command:
+If you are using the macOS operating system with the ZSH terminal shell, make sure that the `zsh-completions` package is installed. With the [brew](https://brew.sh) package manager for macOS, you can check to see if the `zsh-completions` package is installed with the following command:
```

Comment:
Also, this should be Zsh or Z-shell.

Reply:
I originally had it as zsh but the lint test failed hence why I changed it to ZSH, like it is in other places on the page 😅 I will try changing it to Z-shell or Zsh

Reply:
https://www.zsh.org has Zsh which is why I suggested that. You may need to add that to the .spelling file. I suspect the checker allows some acronyms so that's maybe why ZSH passes the check. And I would change the other places in the file in the same way.

Reply:
Ah alright, yeah sounds good, I'll change the other places and add it to .spelling, I did think ZSH in all caps looked odd